snafu/utils/get_prometheus_data.py

```diff
@@ -13,29 +13,30 @@
 class get_prometheus_data():
     def __init__(self, action):
 
+        self.sample_info_dict = action
         self.uuid = action["uuid"]
         self.user = action["user"]
         self.cluster_name = action["cluster_name"]
         self.test_config = action["test_config"]
 
         # change datetime in seconds string to datetime object
-        starttime = datetime.fromtimestamp(int(action["starttime"]))
-        self.start = starttime.datetime()
+        starttime = datetime.fromtimestamp(int(self.sample_info_dict["starttime"]))
+        self.start = starttime
 
         # change datetime in seconds string to datetime object
-        endtime = datetime.fromtimestamp(int(action["endtime"]))
+        endtime = datetime.fromtimestamp(int(self.sample_info_dict["endtime"]))
         # add 120s buffer to end time
-        endtime = endtime + timedelta(seconds=120)
-        self.end = endtime.datetime
+        # endtime = endtime + timedelta(seconds=120)
+        self.end = endtime
 
         # step value to be used in prometheus query
-        self.T_Delta = 30
+        self.T_Delta = 15
 
         self.get_data = False
         if "prom_token" in os.environ and "prom_url" in os.environ:
             self.get_data = True
             token = os.environ["prom_token"]
-            self.url = os.environ["prom_url"]
+            self.url = "http://" + os.environ["prom_url"]
```

Comment:
hm, should we not just tell the user to provide the full URL? Do we get in trouble if we pass `http` but the server is on `https`, will the redirect always happen?